### PR TITLE
feat: remove deprecated storage option for loki and GCS

### PR DESCRIPTION
**Pull request by j-zimnowoda**
The big table index storage is deprecated: https://grafana.com/docs/loki/latest/storage/#index-storage
Now the same indexing as for any other backend is used and stored in GCS.
Also the serviceAccount key is supported.

depends-on https://github.com/redkubes/otomi-api/pull/509

Ream more: https://grafana.com/docs/loki/latest/configure/#gcs_storage_config

### CODE REVIEW

File: values-changes.yaml [hunk 1]
```diff
@@ -121,4 +121,6 @@ changes:
       - 'teamConfig.{team}.selfService.team.downloadDockerConfig': 'teamConfig.{team}.selfService.access'
   - version: 19
     deletions:
+      - 'apps.loki.storage.gcs.project'
+      - 'apps.loki.storage.gcs.instance'
```
Comment (ferruhcihan) [inline values-changes.yaml:125]: Should we add these deletions as `version: 20`? It seems that version 19 has already been released. 
[v2.9.0](https://github.com/redkubes/otomi-core/compare/v2.8.0...v2.9.0)

File: values-schema.yaml [hunk 2]
```diff
@@ -2590,10 +2591,6 @@ properties:
                 properties:
                   bucket:
                     $ref: '#/definitions/wordCharacterPattern'
-                  project:
-                    $ref: '#/definitions/wordCharacterPattern'
-                  instance:
-                    $ref: '#/definitions/wordCharacterPattern'
```
Comment (ferruhcihan) [inline values-schema.yaml:2596]: Should we add `serviceAccount` field (according to otomi-api loki.yaml file)?

File: values-changes.yaml (hunk 1 above)
Comment (merll) [reply values-changes.yaml:125]: Yes, we should merge in `main` once more.

File: values-schema.yaml (hunk 2 above)
Comment (merll) [reply values-schema.yaml:2588]: Yes, it should. It is added now, and also to `values-changes.yaml`.

**Review by ferruhcihan — APPROVED** [review]
LGTM

**Pull request merged**